src/test/java/wbooks/userdata/userdata.feature

```diff
@@ -0,0 +1,41 @@
+Feature: get data from specific user
+
+  Background:
+    * url 'https://wbooks-api-stage.herokuapp.com/api/v1'
+
+    * def login = call read('classpath:wbooks/login/login.feature@login')
+    * def authToken = login.access_token
+
+    * def userDataSchema =
+    """
+    {
+      id: '#present',
+      email: '#present',
+      last_name: '#present',
+      first_name: '#present',
+      rents_counter: '#present',
+      comments_counter: '#present',
+      unread_notifications: '##present',
+      image_url: '##present',
+    }
+    """
+
+  Scenario: get user data
+    Given path 'users/1'
+    And header Authorization = authToken
+    When method get
+    Then status 200
+    And match response == userDataSchema
```

Comment:
you can check that the id returned is the same of the user you are searching

Reply:
Added. Its also using a variable for the expected user id now

Reply:
it's ok! but i want you try validate all in the same line!

Reply:
Done adding the validation to the schema 🤔